cmd/tellor/main.go

```diff
@@ -31,6 +31,9 @@ import (
 )
 
 var ctx context.Context
+var cont tellorCommon.Contract
+var acc tellorCommon.Account
+var clt rpc.ETHClient
```

Comment:
Having global variables is another anti pattern and we need to move away from it. It is in the coding guidelines :smiley:  https://github.com/tellor-io/TellorMiner/blob/master/docs/coding-style-guide.md

There are many articles why global variables are bad.